ipython/simulator_adapter_examples/cantera_simulator.ipynb

```diff
@@ -0,0 +1,142 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook demonstrates how to use one of the cantera simulator adapters"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline\n",
+    "\n",
+    "\n",
+    "from tests.common import run_minimal\n",
+    "from t3.common import IPYTHON_SIMULATOR_EXAMPLES_PATH\n",
+    "from t3.main import T3\n",
+    "from t3.simulate.cantera_constantTP_simulator import CanteraSimulatorConstantTP\n",
+    "\n",
+    "from arc.common import read_yaml_file"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# define path that contains the input file and T3 iteration folders\n",
+    "EXAMPLE_DIR = os.path.join(IPYTHON_SIMULATOR_EXAMPLES_PATH, 'cantera_simulator_data')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# read in the input dictionary to use T3 via its API\n",
+    "minimal_input = os.path.join(EXAMPLE_DIR, 'input.yml')\n",
+    "input_dict = read_yaml_file(path=minimal_input)\n",
+    "input_dict['verbose'] = 10\n",
+    "input_dict['project_directory'] = EXAMPLE_DIR"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# create an instance of T3, which stores information used by the Cantera adapter\n",
+    "t3 = T3(**input_dict)\n",
+    "t3.set_paths()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# simulate ideal gas with constant T and P and perform sensitivity analysis wrt to rate constants and ethalpies\n",
+    "observable_list = ['OH', 'H']\n",
+    "CanteraSimulator_adapter = CanteraSimulatorConstantTP(t3=t3.t3,\n",
```

Comment:
use lower_case_underscores for parameters?

Reply:
Thanks for catching this. I changed all notebooks to use `cantera_simulator_adapter` for example.